include/nccl_ofi.h

```diff
@@ -10,6 +10,7 @@ extern "C" {
 #endif
 
 #include <stdbool.h>
+#include <assert.h>
```

Comment:
We don't need this header for `static_assert`

Reply:
It is required `static_assert is a keyword defined in the <assert. h> header. `

Reply:
Right, this is C, not C++.